transformerlab/routers/jobs.py

```diff
@@ -102,3 +103,21 @@ async def get_training_job_output(job_id: str):
     with open(output_file, "r") as f:
         output = f.read()
     return output
+
+# Templates
+
+
+@router.get("/template/{template_id}")
+async def get_training_template(template_id: str):
+    return await db.get_training_template(template_id)
+
+
+@router.put("/template/update")
+async def update_training_template(template_id: str, name: str,
+                                   description: str,
+                                   type: str,
+                                   config: Annotated[str, Body(embed=True)]):
+    configObject = json.loads(config)
```

Comment:
Possibly you can send bad JSON so it might be nice to catch JSONDecodeError and return an error message.